src/styles/elements/buttons.scss

```diff
@@ -236,8 +236,8 @@
 
     &--special {
         color: #fff;
-        background: linear-gradient(-58deg, #49505c 0%, #3f51b5 36%, #00c7d6 100%);
-
+       // background: linear-gradient(-58deg, #49505c 0%, #3f51b5 36%, #00c7d6 100%);
```

Comment:
Just remove the line. Git history will remember

Reply:
@jsoref , I have pushed the above mentioned changes, so I do not need to create a another PR, right?

Reply:
You do not, if you look at this PR from the files tab, you'll see your changes have been integrated into the PR.

Some projects would want you to squash/fixup your commits as you update, but I'm not sure what this repository's policy is.